<?xml version="1.0"?>
<launch>

<!--     <include file="$(find multi_car_localization)/launch/fake_cars.launch" />

    <node pkg="multi_car_localization" type="fake_vicon.py" name="fake_vicon" output="screen">
        <param name="num_cars" value="3" />
    </node>
 -->

    <group ns="car0">

        <node pkg="multi_car_localization" type="vicon_to_gps_robot_localization.py" name="vicon_to_gps_robot_localization" output="screen">
            <param name="frame_id" value="car0" />
        </node>

	    <node pkg="robot_localization" type="navsat_transform_node" name="navsat_transform_node" respawn="true">
            <param name="frame_id" value="world"/>
            <param name="child_frame_id" value="car0"/>

            <param name="magnetic_declination_radians" value="0"/>

            <param name="yaw_offset" value="0"/>

            <remap from="/imu/data" to="/car0/imu/data" />
            <remap from="/gps/fix" to="/car0/gps/fix" />
            <remap from="/odometry/filtered" to="/car0/odometry/filtered"/>

        </node>

    </group>

</launch>